build_resolvers/lib/src/resolver.dart

```diff
@@ -83,8 +83,9 @@ class AnalyzerResolver implements ReleasableResolver {
   /// later). If this happens we don't want to hide the asset from the analyzer.
   final _seenAssets = new Set<AssetId>();
 
-  AnalyzerResolver(DartUriResolver dartUriResolver) {
```

Comment:
Is this a non-exposed class?

Reply:
Yes, `AnalyzerResolvers` is the only exported class.